docs/application/tizen-studio/native-tools/ui-builder/component-attributes.md

```diff
@@ -546,7 +546,7 @@ A flipselector item component can be placed as a child of the flipselector conta
 
 ### Gengrid (![Gengrid](./media/component_attributes_gengrid_icon.png)) in Mobile Applications
 
-A [gengrid](../../native/guides/ui/efl/component-gengrid-m.md) component displays objects on a grid layout and renders only the visible objects. The gengrid can contain a small number of items.
+A [gengrid](../../native/guides/ui/efl/mobile/component-gengrid.md) component displays objects on a grid layout and renders only the visible objects. The gengrid can contain a small number of items.
```

Comment:
It looks `../../../native/guides/ui/efl/mobile/component-gengrid.md` is correct

Reply:
done